addons/pos_restaurant/models/pos_config.py

```diff
@@ -24,42 +24,8 @@ class PosConfig(models.Model):
 
     def get_tables_order_count_and_printing_changes(self):
         self.ensure_one()
-        floors = self.env['restaurant.floor'].search([('pos_config_ids', '=', self.id)])
-        tables = self.env['restaurant.table'].search([('floor_id', 'in', floors.ids)])
-        domain = [('state', '=', 'draft'), ('table_id', 'in', tables.ids)]
-
-        order_stats = self.env['pos.order']._read_group(domain, ['table_id'], ['__count'])
-        linked_orderlines = self.env['pos.order.line'].search([('order_id.state', '=', 'draft'), ('order_id.table_id', 'in', tables.ids)])
-        orders_map = {table.id: count for table, count in order_stats}
-        changes_map = defaultdict(lambda: 0)
-        skip_changes_map = defaultdict(lambda: 0)
-
-        for line in linked_orderlines:
-            # For the moment, as this feature is not compatible with pos_self_order,
-            # we ignore last_order_preparation_change when it is set to false.
-            # In future, pos_self_order will send the various changes to the order.
-            if not line.order_id.last_order_preparation_change:
-                line.order_id.last_order_preparation_change = '{}'
-
-            last_order_preparation_change = json.loads(line.order_id.last_order_preparation_change)
-            prep_change = {}
-            for line_uuid in last_order_preparation_change:
-                prep_change[last_order_preparation_change[line_uuid]['uuid']] = last_order_preparation_change[line_uuid]
-            quantity_changed = 0
-            if line.uuid in prep_change:
-                quantity_changed = line.qty - prep_change[line.uuid]['quantity']
-            else:
-                quantity_changed = line.qty
-
-            if line.skip_change:
-                skip_changes_map[line.order_id.table_id.id] += quantity_changed
-            else:
-                changes_map[line.order_id.table_id.id] += quantity_changed
-
-        result = []
-        for table in tables:
-            result.append({'id': table.id, 'orders': orders_map.get(table.id, 0), 'changes': changes_map.get(table.id, 0), 'skip_changes': skip_changes_map.get(table.id, 0)})
-        return result
+        # NOTE this method is kept here because we're in stable
+        return {'id': 0, 'orders': [], 'changes': [], 'skip_changes': []}
```

Comment:
That's some big changes in stable to not use a method anymore. I'm not sure it is really stable safe, we should talk about this